_posts/2020-12-03-cloud-native-hpc.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: It's a Bird.. It's a Plane… No! It’s Supercompute in the Cloud!
+date: 2020-12-03
+tags: cloud hpc supercompute lustre aws batch
+image: img/blog/hpc/cloud_computing_datacentre.jpg
+author: Marcelo Aravena
+---
+<center><img src="/img/blog/hpc/cloud_computing_datacentre.jpg" /></center><br/>
+
+
+
+## What is Supercompute aka High Performance Computing (HPC)? 
+HPC, which is also commonly referred to as Supercompute, is a computing platform that aggregates multiple computers into one “Supercomputer”, just like the Voltron Lions join together to make one big robot to achieve greater power and performance in order get the job done quicker, or in Voltrons case; destroy the enemy in a more efficient and effective manner.  HPC technology will help accelerate the processing of your compute intensive workloads, jobs that would normally take days or weeks to complete on a standard single computer can be completed within minutes or hours on a Supercomputer.  
```

Comment:
Is it worth referencing the Woodside press release here around the 1 mill vCPUs?

Reply:
yes, will do it now